index.js

```diff
@@ -531,6 +531,14 @@ function Argv (processArgs, cwd) {
     return argv
   }
 
+  function guessLocale () {
+    try {
+      return osLocale.sync()
```

Comment:
> Shall I release this in the interim to fix the exception?

I think Atom will just pin to version `3.19.0` for now and cut a new release.

Just a heads up, earlier this week we moved `apm` to use `3.19.0` as well because of issues with long paths on Windows introduced by the `os-locale` addition. I know this is a Windows issues, but I just wanted to mention it in case you came across it on that repo.

> what if I add an option .detectLocale(false)?

:+1: That seems good